cassandane/tiny-tests/MailboxVersion/cyrus_indexfile_20

```diff
@@ -0,0 +1,255 @@
+#!perl
+use Cassandane::Tiny;
+use Cyrus::IndexFile;
+use Test::Deep ':v1';
+
+# Ensure v20 index files are right. Note that we don't set an explicit
+# :MailboxVersion(20) here (yet) because this test is testing the *latest*
+# known Mailbox version. When that version changes, this test *should* start
+# failing. When that happens, we should explicitly set :MailboxVersion(20)
+# here, and create a new test (probably copied mostly from this one) for the
+# new version. Repeat to infinity.
+sub test_cyrus_indexfile_20
+{
+    my ($self) = @_;
+    my $jmap = $self->{jmap};
+
+    my $ns_in_a_sec = 1000000000;
```

Comment:
nit: writing `1e9` instead is going to save me from squinting next time I'm here (okay, that's "never" but you get the point)